diff -uNr lorcon2.orig/lorcon_forge.h lorcon2/lorcon_forge.h
--- lorcon2.orig/lorcon_forge.h	2012-08-27 14:42:50.851023843 -0400
+++ lorcon2/lorcon_forge.h	2012-08-27 15:33:49.958191553 -0400
@@ -35,7 +35,7 @@
 
 
 #ifndef __PACKET_ASSEMBLY_H__
-#include <lorcon_packasm.h>
+#include <lorcon2/lorcon_packasm.h>
 #endif
 
 /* Create a random MAC address, optionally seeded with a valid wireless OUI
diff -uNr lorcon2.orig/Makefile.in lorcon2/Makefile.in
--- lorcon2.orig/Makefile.in	2012-08-27 14:42:50.855023542 -0400
+++ lorcon2/Makefile.in	2012-08-27 15:34:08.806283077 -0400
@@ -66,6 +66,7 @@
 	install -m 644 lorcon_packet.h $(INCLUDE)/lorcon2/lorcon_packet.h
 	install -m 644 lorcon_packasm.h $(INCLUDE)/lorcon2/lorcon_packasm.h
 	install -m 644 lorcon_forge.h $(INCLUDE)/lorcon2/lorcon_forge.h
+	install -m 644 lorcon_endian.h $(INCLUDE)/lorcon2/lorcon_endian.h
 	install -m 644 ieee80211.h $(INCLUDE)/lorcon2/lorcon_ieee80211.h
 	install -d -m 755 $(MAN)/man3
 	install -o root -m 644 lorcon.3 $(MAN)/man3/lorcon.3
